ocaml/xapi/helpers.ml

```diff
@@ -1694,6 +1694,19 @@ module Stunnel : sig
 end = struct
   let cert = !Xapi_globs.server_cert_path
 
+  let is_fips_enabled () =
+    Xapi_inventory.lookup ~default:"false" "CC_PREPARATIONS" |> function
+    | "true" ->
+        true
+    | "false" ->
+        false
+    | unknown ->
+        info
+          "CC_PREPARATIONS has unexpected value '%s'. assuming fips mode \
+           should be disabled"
```

Comment:
I would replace "should be" with "is". Otherwise this can create a bit of ambiguity between what we assume and what we do.

Reply:
The function is actually used for both: to determine whether we need to enable FIPS mode, and to determine whether we would have enabled it.

Perhaps we call it cc_mode rather than fips in the name of this function: `is_cc_mode_enabled`, or just `cc_mode`.